pkg/gardenlet/controller/shoot/shoot_control.go

```diff
@@ -280,7 +280,7 @@ func (c *Controller) initializeOperation(ctx context.Context, logger *logrus.Ent
 		WithShootSecretFromReader(gardenClient.Client()).
 		WithProjectName(project.Name).
 		WithExposureClassFromReader(gardenClient.Client()).
-		WithDisableDNS(!seedObj.Info.Spec.Settings.ShootDNS.Enabled).
+		WithDisableDNS(!seedObj.GetInfo().Spec.Settings.ShootDNS.Enabled).
```

Comment:
```suggestion
		WithDisableDNS(!seed.Spec.Settings.ShootDNS.Enabled).
```
I think both are valid but for consistency reasons in this file we can directly use `seed`.

Reply:
Yes, makes sense.

Reply:
Done as suggested.